ngraph/frontend/paddlepaddle/src/op/batch_norm.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "batch_norm.hpp"
+#include <ngraph/opsets/opset6.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                NamedOutputs batch_norm(const NodeContext& node)
```

Comment:
I see activation incorporated into the batch norm [here](https://www.paddlepaddle.org.cn/documentation/docs/zh/1.6/api_cn/layers_cn/batch_norm_cn.html#batch-norm) Am I looking at the wrong spec?

Reply:
Latest Batch Norm doc is https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/static/nn/batch_norm_en.html#batch-norm
Please use PaddlePaddle doc >= 2.1

Reply:
References you've provided is for the Python API -- not for the operations themselves. We should not rely on it.
Here is the better place to see which attributes are included in BatchNorm https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/operators/batch_norm_op.cc#L196

Reply:
Thanks for your reminder, we actually rely on those PDPD cpp codes for attribute extraction. I will use them as PDPD reference links for other ops.